src/library_workerfs.js

```diff
@@ -0,0 +1,95 @@
+mergeInto(LibraryManager.library, {
+  $WORKERFS__deps: ['$FS'],
+  $WORKERFS: {
+    DIR_MODE: {{{ cDefine('S_IFDIR') }}} | 511 /* 0777 */,
+    FILE_MODE: {{{ cDefine('S_IFREG') }}} | 511 /* 0777 */,
+    mount: function (mount) {
+      assert(ENVIRONMENT_IS_WORKER);
+      var root = WORKERFS.createNode(null, '/', WORKERFS.DIR_MODE, 0);
+      (mount.opts.files || []).forEach(function(file) {
```

Comment:
>I think though that people might want a different name

Then it's just `blobs: [{name: "short-name", data: file}]` since `File` is a `Blob`.

>Also I'm not sure the timestamp would be useful.

Some applications might use the mtime. Why not?

>unless I am missing an advantage here

I just thought it would be convenient to get file (or even files) from `<input type=file>` and pass it to WORKERFS right away.

>It seems simpler to me to just accept Blobs

Probably the only advantage is: users won't be confused by two different options?

>if it's documented and tested

It's tested by `test_fs_workerfs_read` and there are some docs. I can add more if it's not enough.